system_strings/sysmes_text.en

```diff
@@ -4258,219 +4331,354 @@ I can't betray her trust.
+// @flowchart_descr
+I wander through the darkness.
+ Hordes of the dead.
+ Countless corpses.
+ This place is hell itself.
 }
 [sha:1569281a96b42d5457c7ad86a4be93cbd5179c43]{
 -- ノエル先生を連れて逃げる。^この異常な空間の中で、^彼女は半狂乱に陥っていた。
-I grab Noel-sensei and escape^with her. In this abnormal^environment, she's turned^half-crazed.
+// @flowchart_descr
+I grab Noel-sensei and escape with her.
+ These freakish catacombs have driven her half-mad.
 }
 [sha:c15e5b067252b15411e52a720a0dbacff30990f7]{
 -- 眠るアルクェイドを置いて、^ひとりマンションを出た。^駅前のデパートへと向かおう。
-I leave the sleeping Arcueid and^exit the apartment alone, heading^for the department store in front^of the station.
+// @flowchart_descr
+I sneak out of the apartment,
+ leaving the still-sleeping Arcueid behind,
+ and head for the department store in front of the station.
 }
 [sha:93795f629b6cf57134f04b2a334b6c7bd4fb3a48]{
 -- 地下に構築された人口の空間。^生け贄を囲む死人の群れ。^どうすれば少女を救い出せる？
-地下に構築された人口の空間。^生け贄を囲む死人の群れ。^どうすれば少女を救い出せる？
+// @flowchart_descr
+There's a man-made chamber underground.
+ A horde of the Dead surrounds the sacrifice.
+ How am I going to save her?
 }
 [sha:a45a7a9dbd8da9a224221f74d276cf23e65324a8]{
 -- 炎が走り抜ける。^それはノエルを焼くばかりか、^彼女の身体を引き込んでいく。
-炎が走り抜ける。^それはノエルを焼くばかりか、^彼女の身体を引き込んでいく。
+// @flowchart_descr
+Fire floods the hallway.
+ Not content to simply burn Noel, he pulls her to him.
 }
 [sha:d3f58df86ea45663206f2650f25c325d16a21368]{
 -- 忘れじの声がする。^見過ごす事はできなかった。^頭で考えるより先に体が動いた。
-忘れじの声がする。^見過ごす事はできなかった。^頭で考えるより先に体が動いた。
+// @flowchart_descr
+I hear a voice I'll never forget.
+ I can't let this to happen.
+ My body starts moving before my mind can catch up.
 }
 [sha:f3235a0598407005b73019ac4cf24dbe7fc370ba]{
 -- ヴローヴはこちらを気にしない。^奇襲は成功するだろう。^このように。
-ヴローヴはこちらを気にしない。^奇襲は成功するだろう。^このように。
+// @flowchart_descr
+Vlov doesn't register me as a threat.
+ I can probably take him by surprise.
+ Just like this――
 }
 [sha:1ebdcf30edfa6f00fd4b5ff535b1ebbbe75bc074]{
 -- 忘れじの声がする。^しかし、ここからでは^あまりにも遠すぎた。
-忘れじの声がする。^しかし、ここからでは^あまりにも遠すぎた。
+// @flowchart_descr
+I hear a voice I'll never forget.
+ But, he's too far away for me to do anything.
 }
 [sha:4166b72f04a1c4be0fae42fd9c431e57300ad3e3]{
 -- このまま押し切って終わらせる。^あと一歩踏み込むだけでいい。^だが、過信の結果は常に非情だ。
-このまま押し切って終わらせる。^あと一歩踏み込むだけでいい。^だが、過信の結果は常に非情だ。
+// @flowchart_descr
+If I keep on going like this I can push through.
+ I just need to take one more step.
+ But the price of overconfidence is steep.
 }
 [sha:8a58d7d6c6b58d4061cb04721f6509bce9567e1a]{
 -- ヤツの能力はホテルで見ている。^ここは一旦距離を取らなければ。^だが、蛮勇の代償は……。
-ヤツの能力はホテルで見ている。^ここは一旦距離を取らなければ。^だが、蛮勇の代償は……。
+// @flowchart_descr
+I got a glimpse of his abilities back at the hotel.
+ I've got to keep my distance.
+ But the price for my recklessness is...
 }
 [sha:757ce7ba8cd98b3e0d46a9b64b66e38441b77772]{
 -- シエル先輩は失敗した。^吹き荒れる白い呪い。^広間は完全に凍り付く――
-シエル先輩は失敗した。^吹き荒れる白い呪い。^広間は完全に凍り付く――
+// Not really a 'failure' so much as the ole vlov cold/hot mistake
+// @flowchart_descr
+Ciel-senpai has made a mistake.
+ A tempestuous white curse.
+ The underground chamber is frozen solid.
```

Comment:
First line is a reference to Offset 11833
>Ciel-senpai messed up.

is what it is there. I don't care either way, but they should be the same.

This is Vlov so might as well get some theming in.
>A billowing white curse.
>...